src/spikeinterface/postprocessing/principal_component.py

```diff
@@ -385,12 +386,15 @@ def _fit_by_channel_local(self, n_jobs, progress_bar):
 
         tmp_folder = p["tmp_folder"]
         if tmp_folder is None:
-            tmp_folder = "tmp"
-        tmp_folder = Path(tmp_folder)
+            if n_jobs > 1:
+                import tempfile
+
+                tmp_folder = tempfile.mkdtemp(prefix="tmp", dir=get_global_tmp_folder())
```

Comment:
Why do we need the prefix tmp? If we are adding a prefix should it be something more informative like principial_component?

Reply:
Oups! Yes! Let me fix both!

Reply:
done, using `pca`

Reply:
Thanks.